uiowa/uiowa
# [Updates] Add `x.com` link support for OEmbed

# How to test


1. pull patch, run `ddev composer install`
2. Sync sandbox site
3. open a page and try to embed this post (https://x.com/drupal/status/1787587442851369230)
4. observe the `x.com` links now work.
